ui/components/stint_tracking/widgets/TableControls/helpers/_on_tracker_stopped.py

```
@@ -0,0 +1,5 @@
+"""Update UI state when tracker stops."""
+
+def _on_tracker_stopped(self) -> None:
+  self._apply_tracking_state(False)
```

Ruff (0.15.5)

[warning] Boolean positional value in function call

(FBT003)